frontend/src/App.tsx

```diff
@@ -985,7 +985,7 @@ export class App extends PureComponent<Props, State> {
               />
               <MainMenu
                 sharingEnabled={this.state.sharingEnabled === true}
-                isServerConnected={this.isServerConnected}
+                isServerConnected={this.isServerConnected()}
```

Comment:
Just checking - will the MainMenu be properly rendered if the server is disconnected while it's showing? (I assume that's why this was passed as a function, and not a value, before?)

Reply:
(I just tested it - looks like it works as expected!)

Reply:
Yes. It will work. Before a function was used cause the underlying options were components, but now they are just objects.